server.js

```diff
@@ -147,7 +151,10 @@ const methods = {
         var fpath = upath.join(working_dir, rpath);
 
         if (fs.existsSync(fpath)){
-            opn(upath.resolve(fpath), { app: editor });
+
+            if (process.env['DISPLAY']){
```

Comment:
Should not be needed. In headless mode the `open-in-editor` config option can be omitted, which makes this code path unavailable.

Reply:
I believe this was because the docker image used had EDITOR set which caused extra inaccessible processes to be started.  I'm probably going to change DISPLAY for a docker specific env.  There are a couple to work with.